ramalama/huggingface.py

```diff
@@ -1,7 +1,12 @@
 import os
-from ramalama.common import run_cmd, exec_cmd
+from ramalama.common import run_cmd, exec_cmd, perror
 from ramalama.model import Model
 
+missing_hugginface="""
+Huggingface models requires the huggingface-cli and tldm modules.
+These modules can be installed via:
+pip install huggingface_hub[cli] tldm
```

Comment:
I don't think we need tldm, huggingface_hub[cli] should be fine

Reply:
```
huggingface-cli: command not found
```

would have been fine too. A user may want to install via pip, pip3, pipx, dnf, apt. Maybe we should let them decide how to install. Some people don't recommend pip.

Reply:
The issue is we don't know if those packages exists via other means.  I can add, 
`or modules might be available via pip3, pipx, dnf, apt or other install tools.`

Reply:
approved nonetheless :)